packages/perspective-jupyter/src/less/index.less

```diff
@@ -15,11 +15,22 @@ div.PSPContainer {
     height: 520px;
 }
 
+// override for small style
+.jp-CodeCell.jp-mod-outputsScrolled .jp-Cell-outputArea {
+    max-height: unset;
+    margin: auto;
+}
+
 // Widget height for Voila
 .jp-OutputArea-output div.PSPContainer {
     height: 520px;
 }
 
+// Widget height for Jupyter Notebook
+.jupyter-widgets-view div.PSPContainer {
+    height: 520px;
+}
+
```

Comment:
???

Reply:
Different CSS rules in notebook so this change is symmetric to the lab CSS rules next to it in the same file.